backend/s3_backend.go

```diff
@@ -142,8 +144,13 @@ func (s *S3Backend) ListFiles(db, version string) ([]string, error) {
 		return nil, s.s3error(err)
 	}
 
-	sort.Strings(res)
-	return res, nil
+	sorted := make([]string, 0, len(res))
```

Comment:
Kinda amused that this is called sorted even before it's sorted, but it's being created in order to be sorted, so fine.